numbagg/decorators.py

```diff
@@ -48,21 +48,22 @@ def ndmoving(*args, **kwargs):
     sum::
 
         @ndmoving([
-            (float64[:], int64, float64[:]),
+            (float64[:], int64, int64, float64[:]),
         ])
-        def move_sum(a, window, out):
+        def move_sum(a, window, min_count, out):
             for i in range(a.size):
                 for j in range(window):
-                    if i - j > 0:
+                    if i - j > min_count:
                         out[i] += a[i - j]
-
-    The default validator for the window argument is between 1 and the array 
-    length. Pass a function to ndmoving as `window_validator` for an 
-    alternative validator
     """
     return _nd_func_maker(NumbaNDMoving, *args, **kwargs)
 
 
+def ndmovingexp(*args, **kwargs):
+    """N-dimensional exponential moving window function."""
+    return _nd_func_maker(NumbaNDMovingExp, *args, **kwargs)
```

Comment:
_Potentially_ we could cut out a layer for these and make the `NumbaNDMovingExp`-like classes decorator themselves (but low payoff and I think this is internal so we have future flexibility)

Reply:
Yes, agreed.